### PR TITLE
Arybka java8test

### PR DESCRIPTION
**Pull request by randrii**
- replaced StringBuilder with String
- equality is true

### CODE REVIEW

File: src/main/java/task/DataGenerator.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import com.github.javafaker.Faker;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class DataGenerator {
+
+    private DataGenerator() {
```
Comment (AqaCoreSchool) [inline src/main/java/task/DataGenerator.java]: whats the purpose of this class if it have private constructor?
Only to hold 1 static method?

File: src/main/java/task/Main.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+public class Main {
+
+    private static final String TEMPLATE = "Name: %s, salary: %s, balance: %s";
+
+    public static void main(String[] args) {
+        List<Person> people = DataGenerator.generatePeople();
+        List<Person> peopleWithSuspiciousBankAccounts = new ArrayList<>();
+
+        people.stream()
```
Comment (AqaCoreSchool) [inline src/main/java/task/Main.java]: why all this logic is in main method? It is really hard to read. Should split it to several private methods

File: src/main/java/task/Main.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                .filter(person -> person.getBankAccount().isActive()
+                        && person.getBankAccount().getBalance() > person.getSalary())
+                .filter(person -> getIncomeSinceAccountStartDate(person) > person.getBankAccount().getBalance())
+                .forEach(peopleWithSuspiciousBankAccounts::add);
+
+        for (int i = 0; i < people.size(); i++) {
+            Person person = people.get(i);
+            if (person.getBankAccount().isActive() && person.getBankAccount().getBalance() > person.getSalary()) {
+                if (getIncomeSinceAccountStartDate(person) <= person.getBankAccount().getBalance()) {
```
Comment (AqaCoreSchool) [inline src/main/java/task/Main.java]: if (getIncomeSinceAccountStartDate(person) <= person.getBankAccount().getBalance())
continue;
why not vice versa?

File: src/main/java/task/DataGenerator.java (hunk 1 above)
Comment (randrii) [reply src/main/java/task/DataGenerator.java]: Fixed

File: src/main/java/task/Main.java (hunk 3 above)
Comment (randrii) [reply src/main/java/task/Main.java]: Fixed

File: src/main/java/task/Main.java (hunk 2 above)
Comment (randrii) [reply src/main/java/task/Main.java]: Fixed

**Pull request closed without merging**